docker-build/Makefile

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2022 Eugene Brodsky (https://github.com/ebr)
```

Comment:
@lstein: is this copyright compatible with the repositorys license?

Reply:
I saw individual contributor's copyright inclusions e.g. [elsewhere in the repo](https://github.com/invoke-ai/InvokeAI/blob/main/scripts/preload_models.py#L2-L7), so figured it would be fine to add. Either way, removed in both places :)